src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -702,17 +702,17 @@ public NamedOnnxValue GetNamedOnnxValue()
     /// | Does this estimator need to look at the data to train its parameters? | No |
     /// | Input column data type | Known-sized vector of <xref:System.Single> or <xref:System.Double> types |
     /// | Output column data type | As specified by the ONNX model |
-    /// | Required NuGet in addition to Microsoft.ML | Microsoft.ML.OnnxTransformer (always),  Microsoft.ML.OnnxRuntime.Gpu (only if GPU processing is used) |
+    /// | Required NuGet in addition to Microsoft.ML | Microsoft.ML.OnnxTransformer (always),  either Microsoft.ML.OnnxRuntime 1.2.0 (for CPU processing) or Microsoft.ML.OnnxRuntime.Gpu 1.2.0 (only if GPU processing is desired) |
     /// | Exportable to ONNX | No |
     ///
-    /// Supports inferencing of models in ONNX 1.2 and 1.3 format (opset 7, 8 and 9), using the
-    /// [Microsoft.ML.OnnxRuntime](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime/) library.
+    /// Supports inferencing of models in ONNX 1.6 format (opset 11), using the
+    /// [Microsoft.ML.OnnxRuntime](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime/) library (version 1.2.0).
     /// Models are scored on CPU by default. If GPU execution is needed (optional), use the
-    /// NuGet package available at [Microsoft.ML.OnnxRuntime.Gpu](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.Gpu/)
-    /// and download [CUDA 9.1 Toolkit](https://developer.nvidia.com/cuda-downloads) and [cuDNN](https://developer.nvidia.com/cudnn).
+    /// NuGet package available at [Microsoft.ML.OnnxRuntime.Gpu](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.Gpu/) (version 1.2.0) instead of the Microsoft.ML.OnnxRuntime nuget (which is for CPU processing)
```

Comment:
Should we say somewhere that the GPU needs to be CUDA supported?

Reply:
Well, I don't think it's necessary since the instructions already mention that CUDA is necessary. Do you think I should mention explicitly that the GPU needs to be CUDA supported?

Reply:
If I am working on my current mac machine (which has an Intel GPU, not compatible with CUDA as far as I know) I might read these instructions and not realize the compatibility issues (until I try and install CUDA), since we advertise ML.NET as cross platform.

Reply:
Perhaps we could add a link to https://developer.nvidia.com/cuda-gpus#compute? @antoniovs1029?

Reply:
I see. So should I include a link to the list of CUDA's supported GPUs (https://www.geforce.com/hardware/technology/cuda/supported-gpus?field_gpu_type_value=All&page=1 ) Or would it be enough to mention that it will only work for GPUs that support CUDA?

Reply:
Thanks, I've added that link. Please let me know if you think there's more I need to change.